js/clients/lib/BcoinClient.js

```diff
@@ -19,12 +25,11 @@ const {utils, BTCUtils} = require('../js/dist');
  */
 class BcoinClient extends NodeClient {
   /**
-   * bcoinclient constructor.
+   * bcoinclient constructor
+   * NB: none since constructor doesn't do anything
+   * (see: eslint-no-useless-constructor)
    * @param {object} options
    */
-  constructor(options) {
```

Comment:
It will automatically call the `NodeClient` constructor and pass along the arguments? Didn't know that

Reply:
If there isn't one specified, it provides a default one. For derived classes:

```
constructor(...args) {
  super(...args);
}
```

I don't like unnecessary code, so it's kind of nice. If we decide we want to put them in at a later time then we can always disable the rule and add them back in.